app/controllers/wpcc/engine_controller.rb

```diff
@@ -3,7 +3,7 @@ class EngineController < Wpcc.parent_controller.constantize
     protect_from_forgery
 
     layout 'wpcc/engine'
-    before_action :log_session
+    before_action :log_session, :check_wpcc_session
```

Comment:
Hi @margOnline is there credible reason for including _wpcc_ in the method names? Is it really needed?

`check_wpcc_session` --> `check_session'